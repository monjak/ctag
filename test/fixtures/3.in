2 Okina, temple to the grandfathers
