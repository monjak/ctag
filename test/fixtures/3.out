2x [c]Okina, temple to the grandfathers[/c]
